src/Spectre.AngularClient/src/main.ts

```diff
@@ -24,6 +24,7 @@ import { environment } from './environments/environment';
 
 if (environment.production) {
   enableProdMode();
+  console.log = function(){};
```

Comment:
What about `console.debug()`?

Reply:
We haven't used debug yet but I can add.